fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AccountingScenarioIntegrationTest.java

```diff
@@ -802,10 +802,10 @@ public void checkPeriodicAccrualAccountingTillCurrentDateFlow() throws Interrupt
 
         float totalInterest = (float) loanSchedule.get(1).get("interestOriginalDue");
         DecimalFormat numberFormat = new DecimalFormat("#.00", new DecimalFormatSymbols(Locale.US));
-        float INTEREST_4_DAYS = totalInterest / totalDaysInPeriod * 4;
-        INTEREST_4_DAYS = Float.valueOf(numberFormat.format(INTEREST_4_DAYS));
+        float interest4Days = totalInterest / totalDaysInPeriod * 4;
+        interest4Days = Float.valueOf(numberFormat.format(interest4Days));
```

Comment:
I don't understand what this does (why it's needed) - but it has nothing to do with the Checkstyle change..

Reply:
LocalVariableName would only allow the name to have following type of regex: ^[a-z][a-z0-9][a-zA-Z0-9]*$|[a-z]

Reply:
No I just meant the Float value format thing in the 2nd line that looks weird, to me.